### DISCUSSION TITLE
Section 5 - Bonus

### DISCUSSION THREAD

**Discussion by noah-black1** [cooperrc/computational-mechanics#232] [General]
How will Section 5 affect our grade? Is it purely extra credit that will be added to our total?

**Comment by Ewicko** [comment]
I was wondering this as well, thanks for making a post here

**Comment by cooperrc** [comment]
Module_05 is purely bonus points e.g. projects are 40% of grade so 100% on Project 05 results in 50/40 on final grade. 

One way to approach this, is 
- if you like trying new things: do the Module_05 work and get bonus points
- if you like diving deeper into previous topics: revise previous work and get points back

both ways should be a way to get the grade you're aiming for
